.gitignore

```diff
@@ -3,4 +3,6 @@
 
 __pycache__/
 *.py[cod]
-*$py.class
\ No newline at end of file
+*$py.class
+
```

Comment:
Please remove this empty line from the `.gitignore`.

Reply:
Ok. What about the empty line on line 3? Is it some kind of grouping?

Reply:
Oh, didn't remember that. Mobile GitHub doesn't let me easily expand a diff but yes there is some grouping. Try to follow that or just ignore it TBH.